backends/eopkg/eopkgBackend.py

```
@@ -272,126 +281,165 @@ def _pkg_to_id(self, pkg):
+                    )
+
+                    # (version, release, build, distro, distro_release)
+                    # If version is different, we should probably compare versions.
+                    # But if they are the same, release comparison is sufficient.
+                    # In Solus/eopkg, release is the primary versioning for updates.
+                    if v_inst[0] == v_avail[0]:
+                        if int(v_avail[1]) > int(v_inst[1]):
+                            candidates.append((available, INFO_AVAILABLE, repo))
+                        elif int(v_avail[1]) == int(v_inst[1]):
+                            # Same version, show both
+                            data = "installed:{}".format(repo) if repo else "installed"
```

Ruff (0.15.12)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)